ui/components/app/transaction-detail/transaction-detail.component.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+
+import { ETH } from '../../../helpers/constants/common';
+import { GasFeeContextProvider } from '../../../contexts/gasFee';
+import { renderWithProvider } from '../../../../test/jest';
+import configureStore from '../../../store/store';
+
+import TransactionDetail from './transaction-detail.component';
+
+process.env.EIP_1559_V2 = true;
```

Comment:
Looks like we might not need this since we have it in the `beforeEach`/`afterEach`?

Reply:
Fixed